src/test/kotlin/com/hibob/academy/dao/PetDaoTest.kt

```diff
@@ -98,8 +100,8 @@ class PetDaoTest @Autowired constructor(private val sql: DSLContext) {
         val pet2 = petCreationRequest2.extractToPet(id2)
         petDao.adoptPet(pet1, ownerId)
         petDao.adoptPet(pet2, ownerId)
-        val updatedPet1 = pet1.copy(ownerId = ownerId)
-        val updatedPet2 = pet2.copy(ownerId = ownerId)
+        val updatedPet1 = petDao.getAllPetsByCompanyId(companyId)[0]
```

Comment:
It is better to call petDao.getAllPetsByCompanyId(companyId) into a val and get [0] and [1] from it istead of calling all twice

Reply:
You right, I changed it in the batch task methods and missed that method. Thanks